ChangeLog.d/fix-tls12server-sent-sigalgs.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+    * Fix a bug whereby the first element of the list of signature algorithms
+      sent as part of the TLS1.2 server certificate request would get
+      overwritten by the list length, leaving two random bytes in the last
+      entry. Found by Simon Nachtigall <simon.nachtigall@achelos.de>.
```

Comment:
Ideally should describe the impact of the bug for the user, rather than the details of what caused the bug. (Also note 'TLS 1.2' should have a space).